README.md

```diff
@@ -25,6 +27,10 @@ config :pool_max, :validate => :number, :default => 50
 # Max number of concurrent connections to a single host. Defaults to 25
 config :pool_max_per_route, :validate => :number, :default => 25
 
+# Specify this if you want to keep the TCP connection open between requests
+# using HTTP Keep-Alive. Defaults to 0 (off)
+config :keepalive, :validate => :number, :default => 0
```

Comment:
I think my confusion is mostly that I don't know (and don't see in the manticore code) what `:keepalive` means as a number. I see it used in two places in Manticore:

* https://github.com/cheald/manticore/blob/ca857e63c717aae4f91bf66f61643dd14a988ee5/lib/manticore/client.rb#L178-L183 - when setting the connection-reuse strategy
* https://github.com/cheald/manticore/blob/ca857e63c717aae4f91bf66f61643dd14a988ee5/lib/manticore/client.rb#L467 - when setting the "Connection: Keep-Alive" header

It's only used as a boolean-ish value in these cases. In the manticore specs, it's only used as a boolean:
https://github.com/cheald/manticore/blob/ca857e63c717aae4f91bf66f61643dd14a988ee5/spec/manticore/client_spec.rb#L557-L593

Thoughts? Maybe time to ask cheald what a Fixnum value actually is intended to mean?

My concern is mostly that we have what I read as a boolean setting "keepalive on or off" but the validation is a number, so 0 means off and nonzero means on and also perhaps some other mysterious property that is uncertain ;)